README.md

```diff
@@ -59,6 +59,16 @@ To get this app to work for you, you will need to have a Trello account and stor
   * BOARD_ID  -  The ID of the board you want to use for the project
 ```
 
+For this application, there are status column headings set in the .env as follows:
+
+```bash
+  * COL_1="Not Started"
+  * COL_2="In Progress"
+  * COL_3="Completed"
+```
+
+These differ from the Trello standard names and can be updated as needed.
+
```

Comment:
Nice spot on making these clear here :) We should also have a section on how to run the tests themselves, e.g. under "Running Tests" it specifies the folder but not the actual command you expect people to run

Reply:
Done